code/modules/projectiles/projectile.dm

```diff
@@ -743,3 +760,23 @@
 
 /obj/projectile/experience_pressure_difference()
 	return
+
+///Like [/obj/item/proc/updateEmbedding] but for projectiles instead, call this when you want to add embedding or update the stats on the embedding element
+/obj/projectile/proc/updateEmbedding()
+	if(!shrapnel_type || !islist(embedding) || !LAZYLEN(embedding))

```

Comment:
Can `embedding` not be a list? A variable should have only one type, what is going on here?

Reply:
Also, can you `null` the `embedding` list once you are done loading it? Or will you still need it afterwards?

Reply:
the !islist must be a carryover from when it could also be a datum, i'll fix that. As for the embedding list, it can still be used in the future if something modifies its stats (like if a trait gives someone a +10 to embed chance with projectiles or items, it modifies the embedding list then calls updateEmbedding() again. Not great, I know, but this is where we end up after one of the ninja's made me kill the embedding datum